lib/pharos/version_command.rb

```diff
@@ -5,18 +5,31 @@ class VersionCommand < Pharos::Command
     def execute
       puts "pharos-cluster version #{Pharos::VERSION}"
       load_phases
+      load_addons
       puts "3rd party versions:"
-      Pharos::Phases.components.each do |c|
+      phases.each do |c|
         puts "  - #{c.name}=#{c.version} (#{c.license})"
       end
       puts "Addon versions:"
-      Pharos::AddonManager.new([__dir__ + '/addons']).addon_classes.each do |c|
+      addons.each do |c|
         puts "  - #{c.name}=#{c.version} (#{c.license})"
       end
     end
 
+    def phases
+      Pharos::Phases.components.sort_by(&:name)
+    end
+
+    def addons
+      Pharos::AddonManager.new(Pharos::Config.new).addon_classes.sort_by(&:name)
```

Comment:
The fake `new` could be avoided, because the `addon_classes` isn't actually an instance method:

```
Pharos::AddonManager.load_addons(__dir__ + '/addons')
Pharos::Addon.descendants
```

Reply:
Fixed.